package.json

```diff
@@ -9,8 +9,8 @@
     "build": "npm run redoc:build && nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"db/**/*.ts\"",
     "start": "nest start",
-    "dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
+    "dev": "npm run redoc:build && nest start --watch",
+    "start:debug": "npm run redoc:build && nest start --debug --watch",
```

Comment:
If this command is for deploying to lower environments, does it still need `--watch`?

Reply:
The `state:debug` command? I consider the watch flag out-of-scope for consideration. It's a command created automatically by the nest configuration. We're focused on adding a prefix command to build the redoc site to make sure the static site is available.